### ISSUE TITLE
Should this be standalone or merged into Core

### ISSUE THREAD

**Issue by bnvk** [bnvk/ConjurorAPI#2]
@simonv3 check it out, i've been experimenting a bit / refactoring Conjuror towards some of my larger goals. This lil wrapper is a simple HTTP route using Hapi server that returns JSON and has some error handling that turns a [data package like this](https://github.com/digitalfreedom/data-targetedthreats) into [this JSON data](https://digitalfreedom.io/api/targetedthreats) and then load a different URL path like `/api/resellers` to load a different package!

This fits into [my master plan for Conjuror](https://github.com/bnvk/Conjuror/issues/38) that I don't think I've shared with you, yet!

Also see [this issue](https://github.com/bnvk/datapackage-query/issues/1) which I didn't tag you in when I filed it.

**Comment by simonv3** [comment]
I like it. It's pretty lightweight, and much more flexible than just the CLI.

My main concern is whether this will affect my existing .csvs I guess?

**Comment by bnvk** [comment]
> My main concern is whether this will affect my existing .csvs I guess?

No, the REST API aspect of conjuror will not affect existing csv data, it is currently just a "read-only" view, making it accept "writes" in the future *could* happen, but that's far off as permissions are something I don't remotely want to think about right now. 

Yah, I lean towards it being merged at some point too, but that then depends on what shape the Conjuror app (which is now a node module) actually takes. Is it just a shell which plumbs these smaller modules together? I dunno, more dev will determine, I suppose :)

**Comment by simonv3** [comment]
Yeah, I think it's fine to create separate modules, but then they should access a consistent back-end. Maybe the single conjuror app just acts as a coupler?
